code/datums/traits.dm

```diff
@@ -646,6 +646,8 @@ ABSTRACT_TYPE(/datum/trait/job)
 	onAdd(mob/living/owner)
 		if(owner.traitHolder?.hasTrait("atheist"))
 			src.faith_mult = 0.2
+		if (prob(15))
+			owner.see_invisible = INVIS_GHOST
```

Comment:
might as well axe this entirely, this implementation of chaplain see_invis still doesn't work and I'm not sure fixing it would be a good thing lol